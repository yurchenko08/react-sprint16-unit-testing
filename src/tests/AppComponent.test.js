describe("just fake tests for App Component", () => {
  it("1st fake test of App Component", () => {
    expect(true).toEqual(false);
  });
});

// TODO: Your test need to be here instead of fake tests
