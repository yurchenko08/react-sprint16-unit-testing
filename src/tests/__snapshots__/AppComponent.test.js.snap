// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component App Render corectly 1`] = `
<div
  className="App"
>
  <h1>
    Hello World!
  </h1>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
</div>
`;

exports[`Test component App Render hell 1`] = `
<div
  className="App"
>
  <h1>
    Hello World!
  </h1>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
</div>
`;

exports[`Test component App Render hello world 1`] = `
<div
  className="App"
>
  <h1>
    Hello World!
  </h1>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
  <div>
    <h3>
      GitHub User Info
    </h3>
    <ul />
  </div>
</div>
`;
